### ISSUE TITLE
How to handle sampling height / inlet for retrieved ICOS data

### ISSUE THREAD

**Issue by gareth-j** [openghg/openghg#321]
ICOS handles sampling_height as a float, we usually use an an string such as `85m`. At the moment I'm converting their float to our format, does this sound sensible?

**Comment by rt17603** [comment]
This is a good question. For the NOAA height I converted this to a string with a 1dp float (e.g. "10.5m") if this wasn't an explicit integer so I think this is something we could integrate across the board to be consistent. May be best pulling that functionality out into utils? That way we can make sure the same rules are applied to all sites?

For reference, the primary reason for using an integer height was to allow this to more easily map against our associated footprints. However, in the acrg code we did then update this to allow the footprint to have a different "height" than the measurements as the footprints don't need to be so fine-grained (could use "10m" footprint for "9.5m", "12.5m" observations for example). This was incorporated into the "acrg_site_info.json" file as another keyword for a given site so we had that conversion when needed but I don't know if we implemented that in our code.
